mapping_parenting_tech/analysis/prototyping/play_store/get_play_store_apps.py

```diff
@@ -0,0 +1,146 @@
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.13.3
+#   kernelspec:
+#     display_name: 'Python 3.8.12 64-bit (''mapping_parenting_tech'': conda)'
+#     language: python
+#     name: python3
+# ---
+
+# %% [markdown]
+# # Code to extract app ids from [list of] web pages saved from Google Play store
+
+# %% [markdown]
+# Do imports and set file locations
+
+# %%
+from pathlib import Path
+from google_play_scraper import Sort, app, reviews_all
+from tqdm import tqdm
+import re
+import json
+import sys
+
```

Comment:
from mapping_parenting_tech import PROJECT_DIR

Reply:
Committed. Would be good to know what `logging` does